.github/workflows/ci.yaml

```
@@ -11,7 +11,7 @@ on:
 
 jobs:
 
-  unit:
+  unitubuntu:
```

```suggestion
  unit-ubuntu:
```
use dash (`-`) to separate words